### PR TITLE
Question box

### PR DESCRIPTION
**Pull request by jon-arnold**
Angela designed a simple questionnaire box for us that will be used on
the life score and lifestyle score elements.

### CODE REVIEW

File: initiatives/lifescore.md [hunk 1]
```diff
@@ -45,6 +45,17 @@ The formula changes are [here](https://docs.google.com/spreadsheets/d/1qa1RLIl1b
 
 9. As a user who's indicated they have diabetes, I should see the Fasting Plasma Glucose question on my health record input so I can more frequently track my blood sugar.
 
+### Visual Inspiration
+
+Many of the elements aren't frequently asked of the user, so we want to come up with unique ways of asking the user without overwhelming them. A few thoughts:
+
+* **Dynamic Question Boxes**: on the Health Record input, we could have a slot to allow for 1-2 dynamic questions to be presented to the user.
+* **Question Box on Dashboard**: on the Dashboard, a dynamic box appears that asks the user a series of questions. The user may [Skip] a question if they choose, but can continue to answer questions there. This reminds me of how **Facebook** and **LinkedIn** ask you questions programmatically to complete your profile more. This also reminds me of Netflix's onboarding process:
```
Comment (comike011) [inline initiatives/lifescore.md]: ftfy.

File: initiatives/lifestyle-score.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ The metrics and frequency of collection is all found [here](https://docs.google.
 
 * CHORE: Think through all the points options for completing these items. Users should be given points for answering these questions, and especially for completing a full set of them each month.
 
+### Question Box Design Concept
+
+![Exercise](http://f.cl.ly/items/383M0P3s0r1t3f0s2X1C/Question%20%3E%20Exercise.png)
```
Comment (comike011) [inline initiatives/lifestyle-score.md]: I like this view, but is minutes of exercise a reliable gauge? Do we want to add some kind of a gauge for "level of rigor"

**Comment by comike011** [comment]
I'll merge this in after a conversation volley, but in the future (as discussed) I'd like to keep pr's to one topic. So this would be 3 separate ones in the future.

**Comment by jon-arnold** [comment]
Sorry @comike011, my branching is crazy. I'll work on that. I think I'm just making too many docs at once. 

I hate plus sign lists.

Lastly, it's a design concept – we won't be using the question she supplied here. I think it was placeholder on her part. Our questions have a lot more rigor and structure to them. It'll need to be broken out into several user stories, though – we'll need to account for an admin to ask the questions plus a question type selector (mult choice, sliding scale, radio buttons, text area).

**Pull request merged**